packages/dds/tree/src/test/simple-tree/discrepancies.spec.ts

```diff
@@ -0,0 +1,909 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "node:assert";
+import {
+	LeafNodeStoredSchema,
+	MapNodeStoredSchema,
+	ObjectNodeStoredSchema,
+	storedEmptyFieldSchema,
+	ValueSchema,
+	type TreeFieldStoredSchema,
+	type TreeNodeSchemaIdentifier,
+	type TreeStoredSchema,
+} from "../../core/index.js";
+import {
+	defaultSchemaPolicy,
+	allowsRepoSuperset,
+	FieldKinds,
+	type FlexFieldKind,
+} from "../../feature-libraries/index.js";
+import { brand } from "../../util/index.js";
+import {
+	FieldKind,
+	normalizeFieldSchema,
+	SchemaFactoryAlpha,
+	toStoredSchema,
+	isRepoSuperset,
+	getAllowedContentDiscrepancies,
+	type AnnotatedAllowedType,
+	type TreeNodeSchema,
+} from "../../simple-tree/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { createFieldSchema } from "../../simple-tree/fieldSchema.js";
+// eslint-disable-next-line import/no-internal-modules
+import { fieldSchema } from "../feature-libraries/modular-schema/comparison.spec.js";
+// eslint-disable-next-line import/no-internal-modules
+import { LeafNodeSchema } from "../../simple-tree/leafNodeSchema.js";
+// eslint-disable-next-line import/no-internal-modules
+import { findExtraAllowedTypes } from "../../simple-tree/discrepancies.js";
+
+// Arbitrary schema name used in tests
+const testTreeNodeIdentifier = brand<TreeNodeSchemaIdentifier>("tree");
```

Comment:
this seems to be used as the name of schema, not as a full identifier, so branding it as one seems wrong.
```suggestion
const testTreeNodeName = "tree";
```